src/types.rs

```diff
@@ -383,3 +383,37 @@ be_impl!(i64);
 be_impl!(i128);
 be_value!(f32);
 be_value!(f64);
+
+#[cfg(feature = "serialize")]
+#[macro_export]
+macro_rules! be_value_serialize {
+    ($t:ty) => {
+        impl RedbValue for $t {
+            type SelfType<'a> = Self;
+            type AsBytes<'a> = Vec<u8> where Self: 'a;
+
+            fn fixed_width() -> Option<usize> {
+                Some(std::mem::size_of::<$t>())
```

Comment:
This doesn't seem right. Does `bincode::serialize` guarantee that the result is the same length as `size_of`?

Reply:
Nope, and the v2 RCs of `bincode` don't even have support for determining what the size of the value would be after serialization.

Reply:
I see. Well types that return `Some` from `fixed_width()` need to return a slice of the same length from `as_bytes()`